lib/spack/spack/spec.py

```diff
@@ -196,7 +196,7 @@
 _any_version = VersionList([':'])
 
 """Types of dependencies that Spack understands."""
```

Comment:
This docstring is on the wrong line. From http://www.sphinx-doc.org/en/stable/ext/autodoc.html#directive-autoattribute:

> documentation can either be put into a comment with special formatting (using a `#:` to start the comment instead of just `#`), or in a docstring _after_ the definition.

As it is now, the docstring above will apply to `_any_version`, and the docstring for `maxint` will apply to `alldeps`.